### DISCUSSION TITLE
Persona Menu won't overlay a header.

### DISCUSSION THREAD

**Discussion by StevenTCramer** [microsoft/fluentui-blazor#1774] [Q&A]
I can't get my Menu to overlay a Fluent Header.  

![image](https://github.com/microsoft/fluentui-blazor/assets/357219/2f4603bd-c54c-44b4-8d6a-7437a6e11f85)

Using a div it works as expected.

![image](https://github.com/microsoft/fluentui-blazor/assets/357219/b4370da7-932b-48cb-9fab-3bdf3ce2c29b)

So trying to figure out what it is about a header. 

This issue looks similar https://github.com/microsoft/fluentui-blazor/issues/1548

**Comment by StevenTCramer** [comment]
The header class sets a z-index to 10, The menu z-index is set to 9999. If I manually set the z-index to auto I can get it to work as expected.
``` <FluentHeader Class="w-full" Style="z-index: auto" >@Title</FluentHeader>```
 
 I don't know why I should need to do this though, if it is a bug or by design.

**Comment by dvoituron** [comment]
Could you give us more details? An example of code?

I'm using this sample and the result seems correct.

```razor
<FluentButton Id="btnOpen1" OnClick="@(() => open = !open)">
    Open menu
</FluentButton>

<FluentMenu Anchor="btnOpen1" @bind-Open="open">
    <FluentMenuItem>Menu item 1</FluentMenuItem>
    <FluentMenuItem>Menu item 2</FluentMenuItem>
</FluentMenu>

<FluentHeader>
    Header text
</FluentHeader>


@code {
    bool open = false;
}
```

![image](https://github.com/microsoft/fluentui-blazor/assets/8350694/48e4fdd3-a350-44cb-9168-d6b83c90342a)

**Comment by StevenTCramer** [comment]
I'm trying to make a reproducible demo.  Seems it only breaks in my app. Will get you a version at my earliest convenience.
